src/mailroom.py

```diff
@@ -0,0 +1,121 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+from __future__ import print_function
+try:
+    input = raw_input
+except NameError:
+    pass
+
+donor_list = {
```

Comment:
Generally, a Module-scope variable like this should be `ALL_CAPS` in order to signal that it is global.